src/tests/nativeaot/SmokeTests/Dataflow/Dataflow.cs

```diff
@@ -577,7 +580,8 @@ public static void Run()
         {
             // Check we detect these as intrinsics
             IntPtr pBytes = Marshal.AllocHGlobal(Marshal.SizeOf(typeof(ClassWithLayout1)));
-            object o = Marshal.PtrToStructure(pBytes, typeof(ClassWithLayout2));
+            // https://github.com/dotnet/runtime/issues/70072
+            //object o = Marshal.PtrToStructure(pBytes, typeof(ClassWithLayout2));
             Marshal.DestroyStructure(pBytes, typeof(ClassWithLayout3));
```

Comment:
This line needs to be commented out too. You do not want to be calling DestroyStructure on uninitialized memory

Reply:
`ClassWithLayout3` is blittable, so the destroy structure stub is a no-op.

(Notice we also call `PtrToStructure` with `ClassWithLayout2`, not `3`. It's misuse either way and the test relies on the fact that these stubs don't do much. We need to use different type for each call, otherwise we wouldn't notice if one of the intrinsics regresses. I don't expect this to be commented out for long. I just didn't want to fix it because I knew Vitek has a huge change in #70204 that the fix would conflict with and I didn't want to annoy him.)